client/src/App/App.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import './App.css';
+import Home from './pages/Home';
+import List from './pages/List';
+
+class App extends Component {
+  render() {
+    const App = () => (
+      <div>
```

Comment:
You should wrap neighbouring JSX elements in a React Fragment to avoid cluttering your DOM with nodes. See here: https://reactjs.org/docs/fragments.html

Remember to import 'Fragment' from React on line 1 of this file